spec/helpers/application_helper/graph_methods_spec.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe ApplicationHelper::GraphMethods, :type => :helper do
+  describe "#user_opengraph" do
+    context "sample user" do
+      let(:user) { FactoryBot.create(:user,
+                                     profile: { display_name: 'Cunes',
+                                                description: 'A bunch of raccoons in a trenchcoat.' },
+                                     screen_name: 'raccoons') }
+
+      subject { user_opengraph(user) }
+
+      it 'should generate a matching OpenGraph structure for a user' do
+        allow(APP_CONFIG).to receive(:[]).with('site_name').and_return('pineapplespring')
+        expect(subject).to eq(<<-EOS.chomp)
+<meta property="og:title" content="Cunes" />
+<meta property="og:type" content="profile" />
+<meta property="og:image" content="http://test.host/images/large/no_avatar.png" />
+<meta property="og:url" content="http://test.host/raccoons" />
+<meta property="og:description" content="A bunch of raccoons in a trenchcoat." />
+<meta property="og:site_name" content="pineapplespring" />
+<meta property="profile:username" content="raccoons" />
+EOS
```

Comment:
with `<<~EOS` the indentation 
```suggestion
        expect(subject).to eq(<<~EOS.chomp)
          <meta property="og:title" content="Cunes" />
          <meta property="og:type" content="profile" />
          <meta property="og:image" content="http://test.host/images/large/no_avatar.png" />
          <meta property="og:url" content="http://test.host/raccoons" />
          <meta property="og:description" content="A bunch of raccoons in a trenchcoat." />
          <meta property="og:site_name" content="pineapplespring" />
          <meta property="profile:username" content="raccoons" />
        EOS
```